librad/src/meta/entity.rs

```diff
@@ -124,15 +130,70 @@ pub enum HistoryVerificationError {
     EmptyHistory,
 
     #[error("Error at revsion (rev {revision:?}, err {error:?})")]
-    ErrorAtRevision { revision: u64, error: Error },
+    ErrorAtRevision {
+        revision: EntityRevision,
+        error: Error,
+    },
 
     #[error("Update error (rev {revision:?}, err {error:?})")]
     UpdateError {
-        revision: u64,
+        revision: EntityRevision,
         error: UpdateVerificationError,
     },
 }
 
+/// Type representing an entity revision
+pub type EntityRevision = u64;
+
+/// Timestamp for entities and signatures, as milliseconds from Unix epoch
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Deserialize, Serialize)]
+pub struct EntityTimestamp(i64);
```

Comment:
So currently the timestamp is just aesthetic, right? It's not used for any business logic right?

Reply:
It might be useful to separate it into its own module so as not to clutter up this one.